packages/core/src/codewhisperer/util/supplementalContext/utgUtils.ts

```diff
@@ -24,10 +24,12 @@ import { getOpenFilesInWindow } from '../../../shared/utilities/editorUtilities'
 import { getLogger } from '../../../shared/logger/logger'
 import { CodeWhispererSupplementalContext, CodeWhispererSupplementalContextItem, UtgStrategy } from '../../models/model'
 
-type UtgSupportedLanguage = keyof typeof utgLanguageConfigs
+const UTG_SUPPORTEDLANGUAGES: vscode.TextDocument['languageId'][] = ['java', 'python']
```

Comment:
this is failing lint.
```suggestion
const utgSupportedLanguages: vscode.TextDocument['languageId'][] = ['java', 'python']
```